packages/web-new/src/scripts/console.js

```diff
@@ -198,6 +196,15 @@ function enableAPIConversionWithProtocol(editorInstance) {
                 showConsoleError("Please insert a TD which uses MQTT!")
             }
         }
+
+        if (AASTab.checked === true) {
+            if (["mqtt", "mqtts", "http", "https", "coap", "modbus"].some(p => protocolSchemes.includes(p))) {
```

Comment:
Why is this check there, we should not limit it to protocols we are aware of... makes it also difficult to maintain in the future.